lib/data-builder.js

```diff
@@ -67,8 +67,10 @@ function fillDebugData(data, err) {
     if ((p in data)) continue;
     data[p] = err[p];
   }
-  // NOTE err.stack is not an enumerable property
+  // NOTE `err.stack` and `err.message` is not an enumerable property
   data.stack = err.stack;
+  if (err.message)
```

Comment:
We don't check whether `err.message` is set in other `fill` functions, is there any reason why we can't do the same here?

I'd also prefer to set the message among the first items, because JSON serializers preserve the order in which properties were defined.

```js
data.message = err.message;

for (var p in err) {
  if ((p in data)) continue;
  data[p] = err[p];
}

data.stack = err.stack;
```

Could you please check that `err.name` is included in debug-mode `data` too?